frontend/src/app/home/Mint/index.tsx

```diff
@@ -48,14 +54,14 @@ export function Mint({ fetchPet }: MintProps) {
     console.log("MINT PET: ", newName, parts);
     const payload = {
       type: "entry_function_payload",
-      function: `${process.env.NEXT_PUBLIC_CONTRACT_ADDRESS}::main::create_aptogotchi`,
+      function: `${NEXT_PUBLIC_CONTRACT_ADDRESS}::main::create_aptogotchi`,
       type_arguments: [],
       arguments: [newName, parts],
     };
 
     try {
       const response = await signAndSubmitTransaction(payload);
-      await provider.waitForTransaction(response.hash);
+      await aptosClient.waitForTransaction(response.hash);
```

Comment:
that's weird, it should accept an object param
https://github.com/aptos-labs/aptos-ts-sdk/blob/main/src/api/transaction.ts#L105

Reply:
it didn't but i didn't get type error from vscode (maybe because `response.hash` has type `any`)?

Reply:
fixed

Reply:
hmm dont think so.. it should still tells you that you should be providing an object value. I do get error on my env, maybe something with your vs code configurations?

Reply:
<img width="726" alt="Screenshot 2023-10-20 at 4 50 48 PM" src="https://github.com/aptos-labs/aptogotchi/assets/129789810/b1b665aa-ecba-40e3-b334-1a57ab0d3a4b">
yeah i think it's my vscode setting